projects/Mallard/src/hooks/use-config-provider.tsx

```diff
@@ -110,13 +121,23 @@ export const ConfigProvider = ({ children }: { children: React.ReactNode }) => {
         }
     }, [])
 
+    useEffect(() => {
+        async function getIsSSR() {
```

Comment:
Nothing wrong with this approach just doesn't follow the code style elsewhere where we use lambdas

Reply:
Updated to reflect the existing code style